src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java

```diff
@@ -308,5 +309,10 @@ public void onGuiChat(ClientChatReceivedEvent e) {
 			unformatted.equals("You have successfully picked the lock on this chest!")
 			|| (unformatted.startsWith("You received +") && unformatted.endsWith(" Powder")))
 			OverlayManager.powderGrindingOverlay.message(unformatted);
+
+		if (unformatted.equals("ENDER NODE! You found Endermite Nest!") && SBInfo.getInstance().getLocation() != null &&
+		SBInfo.getInstance().getLocation().equals("combat_3")) {
```

Comment:
You check the location already inside EnderNodes.java, you can delete the checks here